### DISCUSSION TITLE
The Settings in SEB Config file "Use SEB Settings file for configuring a client" also asking password while launching the Exam

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#453] [Q&A]
I have set password to encrypt the .seb file from opening with Notepad or any editors. While try to open with Notepad, the configuration file encrypted. It was worked.

But, While I launch the exam, SEB asked to enter the settings password. Then I noted that I have selected "**Use SEB Settings file for Starting an exam**". So that SEB asked password while launching the exam.

Then, I changed the option to "**Use SEB Settings file for configuring a client**" as mentioned in the SEB documents for windows. But, Still, the SEB is asking settings password while launching the exam.

What is the solution for this

Here I attached the screenshot of the SEB Config tool.

![Screenshot from 2022-08-26 12-37-09](https://user-images.githubusercontent.com/111885140/187165603-602bb1d8-3a3f-469f-b834-ae49f2d1b5c5.png)
![Screenshot from 2022-08-26 12-40-16](https://user-images.githubusercontent.com/111885140/187165610-f8bf4dc1-4351-4a76-bce7-b477636af125.png)

**Comment by danschlet** [comment]
If you set a specific settings password, SEB will ask for it. If you're using the option "Use SEB Settings file for configuring a client" and don't set a settings password, then SEB encrypts that configuration file with a default password. But obviously that isn't as secure as when you choose an individual password. The idea of an individual settings password is: The config file is encrypted with that password and cannot be decrypted without knowing that password. You could communicate the password just before starting an on-site exam, then you can be sure no one would have been able to use those settings before that moment. 
But it's still more important to use one of the systems which have an SEB integration and check if an exam is accessed with the correct SEB settings (using the Browser Exam or Config Key). Then it might no longer be relevant if the SEB config file is encrypted or not.
